CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
+## 2.0.3
+  - Declare plugin as not worker safe for both server and client mode. (IO#select is not threadsafe)
```

Comment:
I think server mode is worker safe. Can you add a comment in the code linking to where IO#select is noted as not threadsafe in JRuby (docs somewhere on jruby, if possible?)